code/modules/mining/machine_vending.dm

```diff
@@ -60,7 +62,10 @@
 		new /datum/data/mining_equipment("Luxury Bar Capsule",			/obj/item/survivalcapsule/luxuryelite,								10000),
 		new /datum/data/mining_equipment("Lazarus Injector",			/obj/item/lazarus_injector,											1000),
 		new /datum/data/mining_equipment("1000 Space Cash",				/obj/item/stack/spacecash/c1000,									2000),
-		new /datum/data/mining_equipment("Pizza",						/obj/item/pizzabox/margherita,										200),
+		new /datum/data/mining_equipment("Margherita Pizza",			/obj/item/pizzabox/margherita,										200),
+		new /datum/data/mining_equipment("Mushroom Pizza",				/obj/item/pizzabox/mushroom,										200),
+		new /datum/data/mining_equipment("Meat Pizza",					/obj/item/pizzabox/meat,											200),
+		new /datum/data/mining_equipment("Vegetable Pizza",				/obj/item/pizzabox/vegetable,										200),
```

Comment:
The pizza is the same price, meat should be more expensive than vegetables otherwise this absolutely destroys the balance of the game.

Reply:
miners can have a little powergaming

as a treat